libs/dh/globalization/assets-localization/src/assets/i18n/en.json

```diff
@@ -46,6 +46,92 @@
         "meterNumber": "Meternumber"
       }
     },
+    "secondaryMasterData": {
+      "technicalSpecificationTitle": "Tekniske Specifikationer",
```

Comment:
suggestion: When we don't know the translations for some text, we usually add [EN] at the end, e.g. `<text-in-Danish> [EN]`. Let's do that for these as well.